code/game/objects/items/weapons/twohanded.dm

```diff
@@ -272,3 +272,36 @@
 
 	force = 16
 	throwforce = 25
+
+/obj/item/binoculars
+	name = "binoculars"
+	desc = "Used for long-distance surveillance."
+	icon_state = "binoculars"
+	item_state = "syringe_kit"
+	gender = PLURAL
+	flags = FPRINT | TWOHANDABLE
+	slot_flags = SLOT_BELT
+	w_class = W_CLASS_SMALL
+	var/backup_view = 7
```

Comment:
This won't be fool proof and might screw up if the user's view changes while using binoculars, but at the same time I can't really think of a better way.